### ISSUE TITLE
Generate data fails to run with the shown parameters

### ISSUE THREAD

**Issue by Ion-e** [nion-software/nionswift#1228]
The failed item is on the left of the display panel.

![Image](https://github.com/user-attachments/assets/09213787-cea6-455e-80f5-d0b15db0c4e6)

**Comment by cmeyer** [comment]
Yep. this is not a supported data format. Very low priority, but could be fixed.
